.goreleaser.yml

```diff
@@ -19,7 +19,7 @@ builds:
     ldflags:
       - -X github.com/observiq/stanza/version.GitTag={{ .Tag }}
       - -X github.com/observiq/stanza/version.GitCommit={{ .FullCommit }}
-    no_unique_dist_dir: true
+    no_unique_dist_dir: false
```

Comment:
Do we know how this affects our releases?

Reply:
my assumption is it still just attaches the binaries, just want to confirm. There are already some problems with the naming that I'm trying to fix here https://github.com/observIQ/stanza/pull/591

Reply:
Goreleaser keeps track of their location on disk and will upload them the same way. I suspect the naming is wrong (and has been wrong), so I am ignoring that here. Added a comment to your PR.